test/letters/utils/helpers.unit.spec.js

```diff
@@ -69,6 +68,10 @@ describe('Letters helpers: ', () => {
         expect(getBenefitOptionText(option, 'unavailable', true)).to.be.undefined;
       }, ['monthlyAwardAmount', 'serviceConnectedPercentage']);
     });
+    it('should include the awardEffectiveDate in the text for monthlyAward', () => {
+      const tree = getBenefitOptionText('monthlyAwardAmount', 20, true);
+      expect(tree.props.children[1].props.children[0]).to.contain('The effective date');
```

Comment:
I haven't looked at this at all, but it seems like you should be able to do `SkinDeep.shallowRender(tree)`, but if that's one of the things you tried and didn't work, this seems ok.